### PR TITLE
chore(geo): add geofence management ui docs

### PR DESCRIPTION
**Pull request by thaddmt**
_Issue #, if available:_

_Description of changes:_
- Added Geofence UI documentation

Screen Shot 2022-01-05 at 7 15 05 PM

Screen Shot 2022-01-05 at 7 15 11 PM

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by offlineprogrammer — APPROVED** [review]

File: src/fragments/lib/geo/js/geofences.mdx [hunk 1]
```diff
@@ -6,6 +6,49 @@
...
+
+```bash
+npm install -S maplibre-gl@1 maplibre-gl-js-amplify
+```
+
+> **Note:** Make sure that `maplibre-gl-js-amplify` version `1.2.0` or above is installed.
+
+First, create a map onto which you want to add the geofence management component. See the guide on [creating and displaying maps](https://docs.amplify.aws/lib/geo/maps/q/platform/js/).
+
+Then, use `AmplifyGeofenceControl` to get a new instance of the control and add it to your MapLibre map.
```
Comment (Amplifiyer) [inline src/fragments/lib/geo/js/geofences.mdx:25]:
```suggestion
Then, import `AmplifyGeofenceControl` from "maplibre-gl-js-amplify", create a new instance of this control and add it to your MapLibre map instance.

Your app should look like the following
```

File: src/fragments/lib/geo/js/geofences.mdx [hunk 2]
```diff
@@ -6,6 +6,49 @@
...
+    })
+
+    const control = new AmplifyGeofenceControl({})
+    map.addControl(control);
+}
+
+initializeMap();
+```
+
+![A Geofences Management Application](/images/geofence-management-app.png)
```
Comment (Amplifiyer) [inline src/fragments/lib/geo/js/geofences.mdx:52]: We should clean up this image to show only 2 or 3 geofences with proper names. Also would be good to show both circular and polygon geofences and editing/deleting workflow.

**Review by Amplifiyer — COMMENTED** [review]
Are there any common configurations to the control we should mention here? For complete set of configurations add a pointer to the API docs in maplibre-gl-js-amplify package.

File: src/fragments/lib/geo/js/geofences.mdx (hunk 2 above)
Comment (thaddmt) [reply src/fragments/lib/geo/js/geofences.mdx:54]: The only configuration option for now would be to change the `geofenceCollectionId`, will document that in the API.md.

Will update the example to have one screenshot showing the edit workflow with a circle and one screenshot with the delete workflow for a polygon

**Review by Amplifiyer — APPROVED** [review]

**Pull request merged**